profiles/postgres_log/.env.ftl

```diff
@@ -0,0 +1,3 @@
+COMPOSE_FILE=${config["enabled-engines"]?map(engine -> engine + ".compose.yml")?join(":")}
+POSTGRES_LOG_JDBC_URL=jdbc:postgresql://postgres_log:5432/datasqrl
+WAIT_HOSTS=postgres_log:5432
```

Comment:
Please put the env variables where the other engines define them. We can create a separate task to generalize the env, but right now without env merging behavior we will end up with conflicts for other profiles that bring .env files.

Reply:
Done.